lib/components/base-components/PrimitiveComponent.ts

```diff
@@ -279,7 +279,8 @@ export abstract class PrimitiveComponent<
    * schematic components
    */
   computeSchematicPropsTransform(): Matrix {
-    return compose(translate(this.props.schX ?? 0, this.props.schY ?? 0))
+    // need to find a way to make schX and schY 0 before autolayout and apply them after autolayout
+    return compose(translate(0, 0))
```

Comment:
i talked about this on stream today at the 25m point, drew a diagram

https://x.com/seveibar/status/1879928186630443342

![image](https://github.com/user-attachments/assets/579311dc-4dcc-4d75-b01d-9813ecda1ebb)


https://github.com/tscircuit/core/issues/529